update-component-usage.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+cd packages/web-components
+yarn install
+yarn build
+yarn build-bindings
+cd ../react-components
+yarn install
+yarn build
+cd ../core
+yarn install
+yarn build
+cd ../design-system-dashboard-cli
+yarn report
```

Comment:
Can you update our [CLI readme](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/design-system-dashboard-cli/README.md) for how to run this report?

Reply:
Sure. Forgot I included this script here. :)